modules/client-configuration/pages/clients-rh.adoc

```diff
@@ -6,36 +6,67 @@ This section contains information about registering traditional and Salt clients
 
 
 
-== Set up a {rhel} Client
+== Server requirements
 
-Ensure that your client meets these requirements before you start:
-
-* 8{nbsp}GB RAM or more
-* Two or more physical or virtual CPU cores
-* Access to {rhel} installation and subscription media
-* An LVM or an NFS mount is recommended
+[WARNING]
+====
+You are responsible for arranging access to Red Hat base media repositories and RHEL installation media.
+You must obtain support from either Red Hat or {suse} for all your RHEL systems.
+If you do not do this, you might be violating your terms with Red Hat.
+====
 
-You will need to ensure you provision enough disk space.
-The [path]``/var/spacewalk`` directory contains all mirrored RPMs,  and can take a significant amount of disk space.
-For example, the {rhel}{nbsp}6 x86_64 channels require over 90{nbsp}GB.
+Ensure that your {productname} Server meets the requirements outlined at installation:hardware-requirements.adoc
 
 Taskomatic will use one CPU core, and requires at least 3072{nbsp}MB RAM.
-To ensure that Taskomatic has access to enough memory, open the [path]``/etc/rhn/rhn.conf`` configuration file, and add this line:
+
+To ensure that Taskomatic has access to enough memory, open the [path]``/etc/rhn/rhn.conf`` configuration file, add this line:
```

Comment:
@mcalmer is it just me, or this is incorrect after the discussion I saw at the mailing list?

I guess user should not edit `/usr/share/rhn/config-defaults/rhn_taskomatic_daemon.conf` as that would get overwritten by an update.

What should we do?

Reply:
Yes, but the text say rhn.conf which is ok. But i need to check the code again. I think we have a new configfile which should be used. We keep rhn.conf only for memory as backward compatible solution.

Reply:
/etc/rhn/rhn.conf is correct for setting the taskomatic max memory.